misc/test/examples_test.go

```diff
@@ -247,6 +260,14 @@ func TestAccAwsPyS3Folder(t *testing.T) {
 }
 
 func TestAccAwsPyServerlessRaw(t *testing.T) {
+	cmd := exec.Command("dotnet publish", "app")
+	cmd.Path = path.Join(getCwd(t), "..", "..", "aws-ts-serverless-raw", "app")
+	cmd.Dir = path.Join(getCwd(t), "..", "..", "aws-ts-serverless-raw")
+	err := cmd.Run()
+	if err != nil {
+		fmt.Printf("err: %v", err)
+		return
+	}
```

Comment:
From conversation in Slack I think this needs to be deleted until https://github.com/pulumi/pulumi/pull/4559 is available and can be used instead.

Reply:
deleted